classes/Media.php

```diff
@@ -291,10 +291,16 @@ public static function getMediaPath($mediaUri, $cssMediaType = null)
      *
      * @param mixed $version
      *
-     * @return string
+     * @return array
+     *
+     * @deprecated 1.7.7 jQuery is always included, this method should no longer be used
      */
     public static function getJqueryPath($version = null, $folder = null, $minifier = true)
     {
+        @trigger_error(
+            'Media::getJqueryPath() is deprecated since version 1.7.7.0, jquery is always included',
+            E_USER_DEPRECATED
+        );
```

Comment:
Deprecated ok, but then maybe you should hard code the path to jquery and jquery-migrate just like in `setMedia` ?